modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/notifications/ScheduledPublicationUserNotificationHandler.java

```diff
@@ -42,10 +47,31 @@ protected String getBody(
 		JSONObject jsonObject = _jsonFactory.createJSONObject(
 			userNotificationEvent.getPayload());
 
-		boolean showConflicts = jsonObject.getBoolean("showConflicts");
+		long adminUserId = _getAdminUserId(
+			userNotificationEvent.getCompanyId());
 
 		String body = null;
 
+		if (Objects.equals(userNotificationEvent.getUserId(), adminUserId)) {
```

Comment:
This check should go after the `showConflicts` check. Otherwise, the admin user will always receive the java error notification even if the publish fails due to conflicts.